google-cloud-vision/.gitignore

```diff
@@ -7,3 +7,7 @@ jsondoc/*
 
 # Ignore YARD stuffs
 .yardoc
+
+# IDE settings
```

Comment:
Files specific to the developer environment should be ignored in the developer's configuration. I would prefer these do not leak into the repository. (But I admit that I am likely to continue to lose this argument.)

Reply:
I don't have strong opinions about this, but gapic is currently adding the IDE settings section.